### PR TITLE
able to show change password screen

### PR DESCRIPTION
**Pull request by ketrinvania**
# Description, Motivation and Context 
[MOBILE] CONSTRUCT UI CHANGE PASSWORD SCREEN

# Link to the resources (mockups, change description)  
https://trello.com/c/fBM1Opzv

# What is the destination branch of this PR?
development

# Screenshots [Mandatory]
WhatsApp Image 2023-12-17 at 22 53 02_fb40a8c4

WhatsApp Image 2023-12-17 at 22 56 24_e97192db

WhatsApp Image 2023-12-17 at 22 54 08_17ffc66f

WhatsApp Image 2023-12-17 at 22 55 14_d70aaaeb

### CODE REVIEW

File: lib/screens/change_password_screen.dart [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  bool _passwordVisible = false;
+  bool _confirmPasswordVisible = false;
+
+  bool _isButtonDisabled() {
+    return _currentPasswordController.text.isEmpty ||
+        _passwordError ||
+        _newPasswordController.text.isEmpty ||
+        _confirmPasswordController.text.isEmpty ||
+        _newPasswordController.text != _confirmPasswordController.text ||
+        _currentPasswordController.text == _newPasswordController.text;
```
Comment (josuastg) [inline lib/screens/change_password_screen.dart]: ini udh bener logicnya @ketrinvania bukannya inih disabled kalo `!=` ya ? coba di cek lagi ya

Comment (ketrinvania) [reply lib/screens/change_password_screen.dart]: - yang line 30 kan new passwrd & confirm harus sama kan ya jadi kalo != disabled nya aktif
- yang line 31 itu current pswrd sama new pswrd harus beda kan ya jadi kalo == disable nya aktif

Comment (josuastg) [reply lib/screens/change_password_screen.dart]: ohh gapapa @ketrinvania kita butuh confirm password sama dengan new password

Comment (josuastg) [reply lib/screens/change_password_screen.dart]: di tiket gk gua tulis kan requirementnya

Comment (ketrinvania) [reply lib/screens/change_password_screen.dart]: > ohh gapapa @ketrinvania kita butuh confirm password sama dengan new password

bukan nya confirm password == new password itu button able jo?

Comment (ketrinvania) [reply lib/screens/change_password_screen.dart]: maksud lu yang current password == new password kah?

Comment (josuastg) [reply lib/screens/change_password_screen.dart]: iya yg line 31 @ketrinvania

Comment (ketrinvania) [reply lib/screens/change_password_screen.dart]: Oke gua apus ya logic nyaa

**Review by josuastg — APPROVED** [review]

**Pull request merged**